components/BottomMenu.js

```diff
@@ -90,6 +115,14 @@ export const styles = StyleSheet.create({
         backgroundColor: "#2A2E43",
         bottom: -275
     },
+    insideBuildingContainer: {
+        width: "100%",
```

Comment:
we should probably decide if we want to use percentages (the better option) or numbers for dimensions, again for consistency :)

Reply:
It seems like we have more percentages throughout the code than numbers, better go with that.

Reply:
Agreed!

Reply:
percentage is easier for us